build.gradle

```diff
@@ -237,16 +237,19 @@ dependencies {
         }
     }
 
-    implementation "org.apache.logging.log4j:log4j-to-slf4j:2.22.1"
+    implementation "org.apache.logging.log4j:log4j-to-slf4j:2.23.1"
 
-    // NOTE: 0.1.12 is using jakarta instead of javax
-    implementation "uk.ac.ox.ctl:spring-security-lti13:0.1.11"
+    // Note: spring-security-lti13 does not work with jakarta yet, so we built our own custom version and declare its transitive dependencies below
+//    implementation "uk.ac.ox.ctl:spring-security-lti13:0.1.11"
+    implementation files("libs/spring-security-lti13-0.1.12.jar")
 
     // https://search.maven.org/artifact/org.eclipse.jgit/org.eclipse.jgit
     implementation "org.eclipse.jgit:org.eclipse.jgit:${jgit_version}"
     implementation "org.eclipse.jgit:org.eclipse.jgit.ssh.apache:${jgit_version}"
-    implementation "org.eclipse.jgit:org.eclipse.jgit.http.server:${jgit_version}"
+//    implementation "org.eclipse.jgit:org.eclipse.jgit.http.server:${jgit_version}"
```

Comment:
if it is not needed anymore, please remove it
```suggestion
```

Reply:
As this is not a permanent fix, I decided to leave it it, but I added a comment.